draft-scurtescu-secevent-event-stream-management-api.md

```diff
@@ -147,6 +147,13 @@ parameters for the SET delivery method. The actual delivery method is
 identified by the special key "delivery_method" with the value being a URI as
 defined in [DELIVERY](#DELIVERY).
 
+min_verification_interval
+: An integer indicating the minimum amount of time in seconds that must pass
+in between verification requests. If an Event Receiver submits verification
+requests more frequently than this, the Event Transmitter MAY respond with a
+429 status code. An Event Transmitter MUST NOT respond with a 429 status code
```

Comment:
I think "MUST NOT" is too strong, "SHOULD NOT" is good enough IMO. Tracking request frequency in a distributed environment is tricky, and the implementation may occasionally err on the short side.